workspace.json

```diff
@@ -4895,6 +4973,18 @@
             }
           }
         },
+        "e2e-ci": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "args": "--targetName=web --targetHost=localhost:4200",
+            "commands": [
+              "yarn start {args.targetName} &",
```

Comment:
Since these 3 commands are duplicated all over this file, can we extract them to a file? e.g. `scripts/ci/e2e-ci.sh`?

Reply:
Also, if you do that, can you add a trap to kill the forked yarn start? something like

    trap "pkill -P $$" EXIT